lib/std/os/linux.zig

```diff
@@ -874,6 +874,11 @@ pub const MSF = struct {
     pub const SYNC = 4;
 };
 
+/// Can only be called on 64 bit systems.
+pub fn mseal(address: [*]const u8, length: usize, flags: c_ulong) usize {
```

Comment:
avoid unnecessary dependency on C language extensions:

```suggestion
pub fn mseal(address: [*]const u8, length: usize, flags: usize) usize {
```